src/components/TipsPagination.vue

```diff
@@ -119,15 +121,6 @@ export default {
         this.loadingMoreTips = false;
       }
     },
-    async reloadData() {
-      this.loadingTips = true;
-
-      this.tips = await Util.range(1, this.page)
-        .asyncMap(async (page) => Backend
-          .getCacheTips(this.tipSortBy, page, this.address, this.search, this.blacklist));
```

Comment:
@thepiwo Was it intentional to make these requests one by one instead of doing them at the same time? I don't think that something can go wrong if the client will request them at the same time, except it has 100 pages opened or so. Also, it can be handled by the backend allowing to accept page range.

Reply:
my goal here was to get the order right without much code

Reply:
But could probably be done using Promise all as well